modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -599,6 +599,27 @@ In previous versions of {productname}, when running on Firefox and after pressin
 
 This error has been corrected in {productname} 6.3. When running {productname} 6.3 on Firefox and creating a merge tag using `mceInsertContent`, pressing Enter or Return to generate the desired merge tag moves the insertion point to the end of said newly-created merge tag, as expected.
 
+=== The `autoresize` plugin caused the editor area to infinitely resize when `content_css` was set to `document`.
```

Comment:
=== The `autoresize` plugin caused the editor area to infinitely resize when `content_css` was set to `document`